http_logs/index.json

```
@@ -3,14 +3,23 @@
     {# non-serverless-index-settings-marker-start #}{%- if build_flavor != "serverless" or serverless_operator == true -%}
     "index.number_of_shards": {{ number_of_shards | default(5) }},
     "index.number_of_replicas": {{ number_of_replicas | default(0) }},
-    "index.requests.cache.enable": false
+    "index.requests.cache.enable": false{%- if enable_logsdb %},{%- endif %}
```

This should not be needed if the `,` is placed at the end of line 8.

```
{%- if enable_logsdb %},
```

```suggestion
    "index.requests.cache.enable": false{%- if enable_logsdb %},{%- endif %}
```
```suggestion
    "index.requests.cache.enable": false
```